bleep/src/main/scala/bleep/internal/importBloopFilesFromSbt.scala

```diff
@@ -291,7 +298,12 @@ object importBloopFilesFromSbt {
                 logger.warn(s"Couldn't determine dependencies for $mod: $errMsg")
                 Nil
               case Right(project) =>
-                project.dependencies
+                // this means the version is set in a parent POM. there may also be templating involved. Let's defer to coursier for that.
+                if (project.dependencies.exists { case (_, dep) => dep.version == "" }) {
+                  // slow path
+                  val resolved = Resolve().addDependencies(Dependency(project.module, project.version)).run()
```

Comment:
Can we resolve everything in one go?

def of(...): Either[NeedsResolving, ParsedDependency]
then after we can collect all lefts and resolve those?

Reply:
maybe! I'll try :)